src/lib/layouts/workflow-header.svelte

```diff
@@ -145,6 +144,24 @@
       {/if}
     </AccordionLight>
   {/if}
+  {#if currentDetails}
+    <AccordionLight let:open>
+      <div
+        slot="title"
+        class="flex w-full items-center gap-2 rounded p-2 text-xl"
+      >
+        <Icon
+          name="flag"
+          class="text-indigo-600/80"
```

Comment:
Can we use `text-brand` here instead?

```suggestion
          class="text-brand"
```